fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationConfig.java

```diff
@@ -58,33 +50,21 @@
 @ComponentScan("com.netflix.spinnaker.fiat.shared")
 public class FiatAuthenticationConfig {
 
-  @Autowired(required = false)
-  @Setter
-  private RestAdapter.LogLevel retrofitLogLevel = RestAdapter.LogLevel.BASIC;
```

Comment:
Were there any uses of this?  If so, can we preserve the ability to set the log level here?

Reply:
This is not needed because for retrofit2, the log level can be set through this - https://github.com/spinnaker/kork/blob/master/kork-web/src/main/groovy/com/netflix/spinnaker/retrofit/Retrofit2ConfigurationProperties.java#L30-L31

Reply:
At first glance it looks like there was a way to set a log level specific to FiatAuthenticationConfig, but maybe that's not the case.  If FiatAuthenticationConfig was using a global setting then I agree we're ok.